tests/jest/navigation-prompt/__snapshots__/getUnsavedChangesPromptOptions.test.jsx.snap

```diff
@@ -4,16 +4,25 @@ exports[`getPendingActionPromptOptions generates a function that provides approp
 Object {
   "acceptButtonText": "terraApplication.unsavedChangesPrompt.acceptButtonText",
   "buttonOrder": "rejectFirst",
-  "content": undefined,
+  "content": <FormattedMessage
+    id="terraApplication.unsavedChangesPrompt.singlePromptMessage"
+    values={
+      Object {
+        "promptDescription": <b>
+          Test Prompt
+        </b>,
+      }
+    }
+  />,
   "emphasizedAction": "reject",
   "endMessage": undefined,
   "rejectButtonText": "terraApplication.unsavedChangesPrompt.rejectButtonText",
-  "startMessage": "terraApplication.unsavedChangesPrompt.singlePromptMessage",
+  "startMessage": undefined,
```

Comment:
is this right?

Reply:
yeah, @neilpfeiffer wanted the pending action to be bold when only one pending action was present. I am achieving this by passing the start message via content since the start message only accepts a string & content accepts a node. 

https://github.com/cerner/terra-application/pull/27/files#diff-872e0c65ce6a3f6a4dacfdb62d35c2f2R14-R21